katello/hooks/boot/13-hiera.rb

```diff
@@ -1,4 +1,4 @@
-if module_enabled?('foreman')
+if foreman_server?
```

Comment:
I'm going to hold off on merging this in favor of merging https://github.com/theforeman/foreman-installer/pull/542 first. IMHO that solution is cleaner and you get to deal with the merge conflict.